README.md

```diff
@@ -17,10 +17,42 @@ There you'll find documentation, examples, announcements and more.
 
 ### Configuration
 
-Feedjira configuration options can be set by using a block
+#### Stripping whitespace from XML
+
+Feedjira can be configured to strip all whitespace but defaults to lstrip only:
 
 ```ruby
 Feedjira.configure do |config|
   config.strip_whitespace = true
 end
 ```
+
+#### Follow redirect limit
+
+For fetching feeds, the follow redirect limit defaults to 3 but can be set:
+
+```ruby
+Feedjira.configure do |config|
+  config.follow_redirect_limit = 5
+end
+```
+
+#### Request timeout
```

Comment:
Beside that this example is wrong (Request timeout != strip_whitespace ;) you should only explain how to set configuration once (in a chapter for configuration) and than explain the single configuration options. 
Don't write the whole Config-Block for every config - this will be incomprehensible after a few config options.